R/dataDrivenClusters.R

```diff
@@ -90,11 +90,12 @@ dataDrivenClusters <- function(voxel_df, n_pca = 20, n_umap = 2, n_clust = 2,
   }
 
   # Adding region column
-  if (is.null(region) == FALSE){
+  if (!is.null(region)){
     region_col <- rep(region, nrows(voxel_df))
```

Comment:
This is incorrect. You should not be cbinding a column to a list (it should be to `data_df`).

Reply:
fixed